src/tests/test_jsproxy.py

```diff
@@ -189,3 +190,123 @@ def test_jsproxy_kwargs(selenium):
         )
         == 5
     )
+
+
+import time
+
+startup = """
```

Comment:
```suggestion
ASYNCIO_EVENT_LOOP_STARTUP = """
```

Reply:
Right, this becomes rather important now that I've merged the two files, when it was `test_await_jsproxy.py` it was a bit more clear.